docs/book/src/topics/managedcluster.md

```diff
@@ -214,3 +239,83 @@ Current limitations
 - Only supports Standard load balancer (SLB).
   - We will not support Basic load balancer in CAPZ. SLB is generally
     the path forward in Azure.
+
+## Troubleshooting
+
+1.) If a user tries to delete the machinepool which refers to the last system node pool AMMP webhook will reject deletion, so time stamp never gets set on the AMMP. However the timestamp would be set on the MP and would be in deletion state. To recover from this state create a new MP manually referencing the AMMP, fudge required references and finalizers in MP. In the AMMP remove the owner reference to the old MP, and set it to the new MP. Once the new MP is pointing to the AMMP you can delete the old MP. To delete the old MP remove the finalizers in that object.
+
+Here is an Example:
+
+```yaml
+# MachinePool deleted 
+apiVersion: cluster.x-k8s.io/v1alpha4
+kind: MachinePool
+metadata:
+  finalizers:             # remove finalizers once new object is pointing to the AMMP
+  - machinepool.cluster.x-k8s.io
+  labels:
+    cluster.x-k8s.io/cluster-name: capz-managed-aks
+  name: agentpool0
+  namespace: default
+  ownerReferences:
+  - apiVersion: cluster.x-k8s.io/v1alpha4
+    kind: Cluster
+    name: capz-managed-aks
+    uid: 152ecf45-0a02-4635-987c-1ebb89055fa2
+  uid: ae4a235a-f0fa-4252-928a-0e3b4c61dbea
+spec:
+  clusterName: capz-managed-aks
+  minReadySeconds: 0
+  providerIDList:
+  - azure:///subscriptions/9107f2fb-e486-a434-a948-52e2929b6f18/resourceGroups/MC_rg_capz-managed-aks_eastus/providers/Microsoft.Compute/virtualMachineScaleSets/aks-agentpool0-10226072-vmss/virtualMachines/0
```

Comment:
awesome example, if this is a real susbcription ID please remove it (and ideally rebase)

Reply:
it is a fake subscription ID